src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -501,7 +501,8 @@ class device {
             uint32_t status = parseComputeUnitStat(custat, i, cu_stat::stat);
             uint32_t usage = parseComputeUnitStat(custat, i, cu_stat::usage);
             std::string name = psKernels.at(psk_inst).pkd_sym_name;
-            name += ":scu_" + std::to_string(i - computeUnits.size());
+            //name += ":scu_" + std::to_string(i - computeUnits.size());
```

Comment:
Should we remove this line instead of comment it out?

Reply:
done